examples/server/bench/README.md

```diff
@@ -0,0 +1,64 @@
+### Server benchmark tools
+
+Benchmark is using [k6](https://k6.io/).
+
+##### Install k6 - ubuntu
+```shell
+snap install k6
```

Comment:
FYI, on immutable OS (I'm using Fedora Silverblue), installing a new package is quite complicated. Can you link directly to the [official guide](https://k6.io/docs/get-started/installation/) for more option?

Personally I prefer to do anything in a sandbox environment (like distrobox or docker), I'll do a pass in the future to write docker wrapper for test scripts.